tests/instructions/multibyte/fc_extensions.rs

```diff
@@ -0,0 +1,41 @@
+/// A function that tests the FC multibyte instructions
+/// https://pengowray.github.io/wasm-ops/#fc
+///
+/// At the moment, it dies on validation, as the f32 data type is not implemented
+#[ignore] // we don't support multibyte instructions yet
+#[test_log::test]
+#[should_panic(expected = "i32.trunc_sat_f32_s NOT implemented")]
```

Comment:
Wouldn't this crash since there is no FC operation implemented?

Reply:
Well, that's why I added the `#[ignore]` attribute